docs/source/spec/core/selectors.rst

```diff
@@ -5,59 +5,130 @@ Selectors
 =========
 
 A :dfn:`Smithy selector` is a domain specific language (DSL) used to match
-specific shapes within a model. Selectors are used to build custom
+shapes within a model. Selectors are used to build custom
 :ref:`validators <EmitEachSelector>` and to specify where it is valid to
-apply a trait.
+apply a :ref:`trait <trait-shape-properties>`.
 
 .. contents:: Table of contents
-    :depth: 2
+    :depth: 1
     :local:
     :backlinks: none
 
 
 Introduction
```

Comment:
Great work on the flow of this documentation. Selectors are now a highly complex topic and these docs help reveal that complexity properly.